### DISCUSSION TITLE
What would be the best way to resize?

### DISCUSSION THREAD

**Discussion by sarthakpati** [Bayer-Group/tiffslide#20] [Q&A] [answered]
My original WSI is of size `(15360, 13312)` which I would like to resize to `(3840,3328)` to write quick unit tests. What would be the quickest way to do this via tiffslide?

Thanks!

**Comment by ap--** [comment, marked as answer]
Hi @sarthakpati,

Seems I haven't gotten a notification that a new discussion opened, so here's my answer with a slight delay:

So I assume the goal of all of this is making sure that the specific file type is actually handled throughout the tests correctly. And that can be a bit complicated, because dependent on the pyramidal format you want to run through your test pipeline it's possible that there's no tool that supports writing that format.

If it's just about resizing and storing in some format you could use `pyvips` or even just plain `opencv`. You can also use `tifffile.imwrite` to write tiffs in various flavors. It's unlikely that writing images is going to be supported directly via tiffslide soon, but if it happens, it will likely use `tifffile`.

Cheers,
Andreas 😃

**Reply by sarthakpati** [reply]
Thanks for the clarity, Andreas!
